src/device-config.ts

```diff
@@ -600,13 +600,8 @@ function checkBoolChanged(
 // Modifies conf
 // exported for tests
 export function ensureRequiredOverlay(deviceType: string, conf: ConfigOptions) {
```

Comment:
We should remove the export of this function since it is only there so that tests don't freak out from accessing a private value. Then in the tests add the right error suppression.